piff/basis_interp.py

```diff
@@ -46,51 +46,53 @@ def __init__(self):
         self.degenerate_points = True  # This Interpolator uses chisq quadratic forms
         self.q = None
 
-    def initialize(self, star_list, logger=None):
-        """Initialize the interpolator prefatory to any solve iterations.
-        This class will initialize everything
-        to have constant PSF parameter vector taken from the first Star in the list.
+    def initialize(self, stars, logger=None):
+        """Initialize both the interpolator to some state prefatory to any solve iterations and
+        initialize the stars for use with this interpolator.
 
-        :param star_list:   A list of Star instances to use to initialize.
+        This class will initialize everything to have constant PSF parameter vector taken
+        from the first Star in the list.
+
+        :param stars:       A list of Star instances to use to initialize.
         :param logger:      A logger object for logging debug info. [default: None]
 
         :returns:           A new list of Stars which have their parameters initialized.
         """
+        c = stars[0].fit.params.copy()
+        self.q = c[:,np.newaxis] * self.constant(1.)[np.newaxis,:]
```

Comment:
So all classes based on BasisInterp must have a `constant' function? Maybe a placeholder function should be put in so we know what it needs to be.

(I see this is already in the master branch. I apologize for not catching this earlier!)